modules/nvidia-gpu-features.adoc

```diff
@@ -54,5 +54,5 @@ Up until this point, the GPU Operator only provisioned worker nodes to run GPU-a
 +
 You can configure the GPU Operator to deploy different software components to worker nodes depending on which GPU workload is configured to run on those nodes.
 
-GPU Operator dashboard::
-You can install a console plugin to display GPU usage information on the cluster utilization screen in the {product title} web console. GPU utilization information includes the number of available GPUs, power consumption (in watts) for each GPU and the percentage of GPU workload used for video encoding and decoding.
+GPU Monitoring dashboard::
+You can install a monitoring dashboard to display GPU usage information on the cluster Observe screen in the {product-title} web console. GPU utilization information includes the number of available GPUs, power consumption (in watts), temperature (in C), utilization (in percent), and other metrics for each GPU.
```

Comment:
Nit pick: the unit of temperature is a degree Celsius, so IMO it should say `in degrees Celsius` or `in °C`.

Reply:
Fixed